wqflask/wqflask/markdown_routes.py

```diff
@@ -0,0 +1,38 @@
+"""Markdown routes
+
+Render pages from github, or if they are unavailable, look for it else where
+"""
+import os
+import requests
+import mistune
+
+from flask import Blueprint
+from flask import render_template
+
+glossary_blueprint = Blueprint('glossary_blueprint', __name__)
+
+
+def render_markdown(file_name):
+    """Try to fetch the file name from Github and if that fails, try to
+look for it inside the file system
+
+    """
+    markdown_url = (f"https://raw.githubusercontent.com"
+                    f"/genenetwork/genenetwork2/"
+                    f"wqflask/wqflask/static/"
+                    f"{file_name}")
+    md_content = requests.get(markdown_url)
+    if md_content.status_code == 200:
+        return mistune.html(md_content.content.decode("utf-8"))
+
+    with open(os.path.join(os.path.abspath(os.path.dirname(__file__)),
+                           f"static/markdown/{file_name}")) as md_file:
+        markdown = md_file.read()
+        return mistune.html(markdown)
```

Comment:
Are you using mistune==0.8.4 or below try using mistune>=2.0.0a4 that might 
fix the bug you are having